src/services/handlers/searchResultsApi.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiResponse } from '../../Models/ApiResponse';
+import { geoNetworkClient } from '../../config/geoNetworkClient';
+import { BoolModel, Must, Query, Querystring, SearchRequest } from '../../Models/SearchRequest';
+
+const url = '/search/records/_search?bucket=s101';
+
+const getSearchRequest = function (searchTerm: string | null): SearchRequest {
+  const queryString = new Querystring(searchTerm as string, 'AND');
+  const must = new Must(queryString);
+  const boolModel = new BoolModel([must]);
+  const queryModel = new Query(boolModel);
+  const searchRequestObj = new SearchRequest(queryModel, null);
+  return searchRequestObj;
+};
+
+const getSearchResults = async function (query: string | null): Promise<ApiResponse> {
+  try {
+    const searchRequestObj = getSearchRequest(query);
+    const res = await geoNetworkClient.post(url, searchRequestObj);
```

Comment:
are you sure the relative url works? or i hope the base url should be concatenated from the constant